Helper/ShipmentOptions.php

```diff
@@ -213,25 +213,26 @@ public static function getValueFromAttribute(
     }
 
     /**
+     * @param  array  $options
      * @param  string $key
      *
      * @return bool|null boolean value of the option named $key, or null when not set in $options
      */
-    public function getValueOfOptionWhenSet(string $key): ?bool
+    public static function getValueOfOptionWhenSet(array $options, string $key): ?bool
     {
-        if (! isset($this->options[$key]) || ! array_key_exists($key, $this->options)) {
+        if (! isset($options[$key]) || ! array_key_exists($key, $options)) {
```

Comment:
Als je toch `return null` doet kan je de `array_key_exists` hier achterwege laten volgens mij, wat er werkelijk gebeurt is nu nodeloos complex genoteerd volgens mij. Volgens mij gebeurt er dit: `return isset($options[$key]) ? (bool) $options[$key] : null);` maar dat is mogelijk niet wat je wilt. Ik denk dat je dit bedoelt: `if ($options[$key] || array_key_exists[$key, $options]) { return (bool)$options[$key]; } return null;` maar dan netjes opgeschreven

Reply:
Wanneer isset false teruggeeft, is de eerste conditie true, dus geeft ie null terug. Als isset true teruggeeft is de eerste conditie false, dus dan gaat ie naar array_key_exists kijken, maar die is altijd true, want isset is ook al true.